### ISSUE TITLE
Allow saving a CustomerOrder with a closed ticket as long as the closed ticket isn't affected or changed

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1737]
_Imported from trac ticket 1737.
Created by **kbower**
Opened in trac: Jun 28, 2012, 1:23:30 PM
Last modified in trac: Jul 2, 2012, 5:46:09 PM_
***

**Comment by kbower** [comment]
_**kbower** trac comment on Jun 29, 2012, 8:02:29 AM:_

Rob, you you think we should reject this ticket if it is half to full day (including testing)?

**Comment by kbower** [comment]
_**kbower** trac comment on Jun 29, 2012, 9:06:14 AM:_

(Changeset [[2800]](../commit/79038ba639d0e0b3d4056f725966ac189f9df684))
 * closes #1737, Allow saving of already closed orders as long as the save doesn't change it (this isn't closing orders yet, but saving already closed orders)

**Comment by kbower** [comment]
_**kbower** trac status change on Jun 29, 2012, 9:06:14 AM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac comment on Jul 2, 2012, 5:43:26 PM:_

When the closed ticket had discounts, fatal error when trying to save:
```
Error message:
AttributeError: Either 'OrderDetailDiscount' object has no attribute '_orig' or evaluating '_orig' caused an AttributeError
```

**Comment by kbower** [comment]
_**kbower** trac status change on Jul 2, 2012, 5:43:26 PM:_ **reopened**

**Comment by kbower** [comment]
_**kbower** trac comment on Jul 2, 2012, 5:46:09 PM:_

(Changeset [[2819]](../commit/44adf038f7cd89b4eb2cff6c5da092eee60c52da))
 * fixes #1737 regarding "'OrderDetailDiscount' object has no attribute '_orig'" fatal error

**Comment by kbower** [comment]
_**kbower** trac status change on Jul 2, 2012, 5:46:09 PM:_ **closed**
